test/linear_solver_routines/block_solver_helpers/proc_gentlemans_house.jl

```diff
@@ -0,0 +1,49 @@
+# This file is part of RLinearAlgebra.jl
+
+module ProceduralTestGent
+
+using Test, RLinearAlgebra, LinearAlgebra, Random
+
+@testset "Gent Projection -- Procedural" begin
+    A = rand(10,5)
+    x = rand(5)
+    b = A * x
+    G = RLinearAlgebra.GentData(A, 3)
```

Comment:
Add comment that you are fixing the block size to be 3 equations

Reply:
I have added this comment